x-pack/plugins/enterprise_search/public/applications/workplace_search/components/shared/source_icon/source_icon.tsx

```diff
@@ -21,5 +21,10 @@ interface SourceIconProps {
 }
 
 export const SourceIcon: React.FC<SourceIconProps> = ({ name, serviceType, className, size }) => (
-  <EuiIcon type={images[camelCase(serviceType)]} title={name} className={className} size={size} />
+  <EuiIcon
+    type={images[camelCase(serviceType)]}
+    title={`${name} logo`}
```

Comment:
[not a blocker] you _may_ want to i18n this, or possibly not include a title at all, if the name of the logo is already included in text nearby 🤔